### PR TITLE
Implement Page header

### PR DESCRIPTION
**Pull request by tiborFulop123**

### CODE REVIEW

File: src/components/HeaderComponent.vue [hunk 1]
```diff
@@ -1,29 +1,74 @@
...
-      <h1>Header </h1>
-    </header>
-  </template>
+  <header>
+
+
+    <div class="flex items-center">
+      <span class="inline-block align-middle TodoHeader">
+        To Do List
+      </span><input type="text" v-model="todoText" />
```
Comment (roli-wg) [inline src/components/HeaderComponent.vue]: The header doesn't have any input in the design so please remove unnecessary or unused html tags

File: src/components/HeaderComponent.vue [hunk 2]
```diff
@@ -1,29 +1,74 @@
...
+  </header>
+  <!-- 
+
+    <div>
+      <input type="text" v-model="todoText" />
+      <button class="" @click="addToDo(todoText)">&#43;</button> 
+    </div>
+   -->
+  <!-- 
+   <TodoForm     @toDoAdded="addNewTodo">   </TodoForm> -->
```
Comment (roli-wg) [inline src/components/HeaderComponent.vue]: It is not a good practice to leave commented code in your work once you think that all the functionality and requirements are in place you don't anymore need this commented parts so please make sure to remove all the commented code in all your files

File: src/components/HeaderComponent.vue [hunk 3]
```diff
@@ -1,29 +1,74 @@
...
+import TodoForm from "./TodoForm.vue";
+import pluszGomb from "./../assets/pluszGomb.svg"
+import { ref } from "vue";
+import buttonIcon from "./../assets/buttonIcon.svg"
+const todoText = ref("");
+const emit = defineEmits(["toDoAdded"]);
+function addToDo() {
+  emit("toDoAdded", todoText.value);
+  todoText.value = "";
+}
```
Comment (roli-wg) [inline src/components/HeaderComponent.vue]: Please make sure that you import and declare only the components and variables which you actually using. Try to keep your code clean so please remove all the unnecessary imports, variables and functions if are any

File: src/components/HeaderComponent.vue [hunk 4]
```diff
@@ -1,29 +1,74 @@
...
+//     };
+//   },
+//   components: { TodoForm },
+// };
+
+// const toDos = ref([]);
+// function addNewTodo(todo) {
+//   toDos.value.push(todo);
+// }
+// // const emit = defineEmits(["toDoAdded"])
```
Comment (roli-wg) [inline src/components/HeaderComponent.vue]: Remove all commented code

File: src/components/HeaderComponent.vue [hunk 5]
```diff
@@ -1,29 +1,74 @@
...
-  
\ No newline at end of file
+  color: #000000;
+
+  font-family: "Neue Haas Grotesk Display Pro";
+  font-style: normal;
+  font-weight: 600;
+  font-size: 72px;
+  line-height: 86px;
+}
```
Comment (roli-wg) [inline src/components/HeaderComponent.vue]: Try to use tailwind css for styling whenever you can. 
Please change all this styles with tailwind classes

File: src/components/HelloWorld.vue [hunk 6]
```diff
@@ -12,16 +12,22 @@ function addNewTodo(todo) {
...
 }
+
+
+
 </script>
 
 <template>
-  <TodoForm @toDoAdded="addNewTodo" @toDoCleared="clearToDo"> </TodoForm>
+
+  <div>    </div>
```
Comment (roli-wg) [inline src/components/HelloWorld.vue]: Please remove empty html tags

**Review by roli-wg — CHANGES_REQUESTED** [review]

**Comment by roli-wg** [comment]
Will make an exception and refrain from reviewing code other than the Header task related.

**Comment by roli-wg** [comment]
Please add prettier and use prettier on this project with the following configuration:
`
{
"singleQuote": true,
"useTabs": false,
"tabWidth": 2,
"vueIndentScriptAndStyle": true
}

`

File: src/components/HomePage.vue [hunk 7]
```diff
@@ -0,0 +1,35 @@
+<script setup>
+  import { ref } from 'vue';
+  import ToDoList from './ToDoList.vue';
+  import HeaderComponent from './HeaderComponent.vue';
+  import toDoListMT from './../assets/toDoListMT.svg';
```
Comment (komaromiGyorgy) [inline src/components/HomePage.vue:5]: Can you give a more explicit name to this image? It is not very clear what this image is from the name of it, something like todoListPlaceholder

**Review by komaromiGyorgy — APPROVED** [review]

**Review by roli-wg — APPROVED** [review]

**Pull request merged**